src/net.h

```diff
@@ -1542,9 +1543,9 @@ friend class CNode;
     int epollfd{-1};
 #endif
 
-    /** Protected by cs_vNodes */
-    std::unordered_map<NodeId, CNode*> mapReceivableNodes GUARDED_BY(cs_vNodes);
-    std::unordered_map<NodeId, CNode*> mapSendableNodes GUARDED_BY(cs_vNodes);
+    Mutex cs_sendable_receivable_nodes;
```

Comment:
there is not really many changes in bitcoin related to this mutex, so, I guess there won't be too much conflicts in future due to that optimization.

But check this commit from future:
```
commit d51d2a3bb5b0011efa1b4f1e2c9512a16ce9b347
Author: Sebastian Falbesoner <sebastian.falbesoner@gmail.com>
Date:   Sat Aug 28 20:57:52 2021 +0200

    scripted-diff: rename node vector/mutex members in CConnman
    
    -BEGIN VERIFY SCRIPT-
    
    ren() { sed -i "s/$1/$2/g" $3 $4 $5; }
    
    ren cs_vAddedNodes         m_added_nodes_mutex     src/net.h src/net.cpp
    ren vAddedNodes            m_added_nodes           src/net.h src/net.cpp
    ren cs_vNodes              m_nodes_mutex           src/net.h src/net.cpp src/test/util/net.h
    ren vNodesDisconnectedCopy nodes_disconnected_copy src/net.cpp
    ren vNodesDisconnected     m_nodes_disconnected    src/net.h src/net.cpp
    ren vNodesCopy             nodes_copy              src/net.cpp
    ren vNodesSize             nodes_size              src/net.cpp
    ren vNodes                 m_nodes                 src/net.h src/net.cpp src/test/util/net.h
```
let's choose name accordingly new style with m_ prefix.

Reply:
Hmm; feel free to propose a commit, but I don't think this is more than a nit

Reply:
will do when backport d51d2a3bb5b0011efa1b4f1e2c9512a16ce9b347